flow.json

```diff
@@ -33,7 +33,6 @@
 			}
 		},
 		"ExampleNFT": "./contracts/core/ExampleNFT.cdc",
-		"ZayVerifierV2": "./contracts/core/ZayVerifierv2.cdc",
```

Comment:
Can we add here the mock signers so we can add the test cases for 20/20 signers?